### DISCUSSION TITLE
请求增加“调整合集列表大小”功能

### DISCUSSION THREAD

**Discussion by thbngy** [the1812/Bilibili-Evolved#3600] [功能建议 / Ideas]
![image](https://user-images.githubusercontent.com/104757355/184809805-bb9cd476-bae3-448c-beb8-1b4ae93ef8ac.png)
弊站给给合集列表的位置也太小了，合集、各分集标题都显示不全，分集条目也只能显示4个，有时候分集多了很难找。
希望能增加显示完整标题+自定义高度宽度之类的功能，感谢！

**Comment by Geeyun-JY3** [comment]
#3380
